src/pages/2020/components/Carousel/styles.module.scss

```diff
@@ -10,41 +10,15 @@
     transition: transform 300ms ease-in-out;
 }
 .slide {
-    width: 80vw;
-    max-width: 1200px;
-    background: $backgroundGradient;
+    margin-right: 6rem;
     border-radius: 0.5rem;
-    cursor: pointer;
     overflow: hidden;
-    flex-shrink: 0;
-    margin-right: 6rem;
-    max-height: 90vh;
-    &.currentSlide {
-        border: 2px solid $twentyTwentyBlue;
-        cursor: url("data:image/svg+xml,%3Csvg width='32' height='32' viewBox='0 0 40 40' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M13.375 33.2739L32.125 22.4486C33.625 21.5825 33.625 19.4175 32.125 18.5514L13.375 7.72612C11.875 6.8601 10 7.94263 10 9.67468L10 31.3253C10 33.0574 11.875 34.1399 13.375 33.2739Z' fill='%23FFC90A' stroke='%231B1C1E' stroke-width='0.5'/%3E%3C/svg%3E%0A"), auto;
-    }
-    .icon {
-        width: 94px;
-        height: 94px;
-        border-radius: 0.5em;
-        background: $darkBlack;
-        padding: 2rem;
-        flex-shrink: 0;
-    }
-    .details {
-        padding: 2rem 2rem;
-    }
-    .imageWrapper {
-        width: 100%;
-        height: 0;
-        padding-bottom: 40%;
-        position: relative;
-    }
-    .image {
-        position: absolute;
-        width: 100%;
-        height: 100%;
-    }
+    cursor: pointer;
+}
+.currentSlide {
+    border: 2px solid $twentyTwentyBlue;
+    box-shadow: 0px 74px 124px 0px rgba(15, 17, 23, 0.48);
+    cursor: url("data:image/svg+xml,%3Csvg width='32' height='32' viewBox='0 0 40 40' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M13.375 33.2739L32.125 22.4486C33.625 21.5825 33.625 19.4175 32.125 18.5514L13.375 7.72612C11.875 6.8601 10 7.94263 10 9.67468L10 31.3253C10 33.0574 11.875 34.1399 13.375 33.2739Z' fill='%23FFC90A' stroke='%231B1C1E' stroke-width='0.5'/%3E%3C/svg%3E%0A"), auto;
```

Comment:
another random thought — what if we had variants for stuff like this

when I was adding the custom li marker for the promotions page, I was thinking that...this could/should be more re-useable, so I wonder if we have a cursor object, and inside are all the variants for different cursors. Similarly we could have an bullet object w/variants for custom bullets....

again — just random thought